FWCore/Framework/interface/global/implementors.h

```diff
@@ -439,6 +442,53 @@ namespace edm {
 
         virtual void accumulate(StreamID streamID, Event const& ev, EventSetup const& es) const = 0;
       };
+
+      template <typename T>
+      class Transformer : public virtual T, private TransformerBase {
+      public:
+        Transformer() = default;
+        Transformer(Transformer const&) = delete;
+        Transformer& operator=(Transformer const&) = delete;
+        ~Transformer() noexcept(false) override{};
+
+        template <typename G, typename F>
+        void registerTransform(ProductRegistryHelper::BranchAliasSetterT<G> iSetter,
+                               F&& iF,
+                               std::string productInstance = std::string()) {
+          registerTransform(edm::EDPutTokenT<G>(iSetter), std::forward<F>(iF), std::move(productInstance));
+        }
+
+        template <typename G, typename F>
+        void registerTransform(edm::EDPutTokenT<G> iToken, F iF, std::string productInstance = std::string()) {
```

Comment:
I decided I don't want to pass std::forward<F> to the lambda in the (highly unlikely) case the lambda is declared outside the function hand held with `auto const&` since the type passed to the lambda would be a reference(?). I definitely want to make it clear the lambda holds a copy.

Reply:
Ok.

Reply:
changed to using `std::move`